src/MakePotCommand.php

```diff
@@ -554,6 +554,16 @@ protected function extract_strings() {
 						'extensions' => [ 'js' ],
 					]
 				);
+
+				MapCodeExtractor::fromDirectory(
+					$this->source,
+					$translations,
+					[
+						'include'    => $this->include,
+						'exclude'    => $this->exclude,
+						'extensions' => [ 'map' ],
```

Comment:
`.css.map` files are a thing, so I would make this more explicit by specifying `.js.map`

Reply:
Unfortunately the `getExtension` method used ( http://php.net/manual/en/splfileinfo.getextension.php ) only returns the final extension.

But I can add an additional check in the `MapCodeExtractor` to ensure we're dealing with JS files.